### ISSUE TITLE
Support for merge keys

### ISSUE THREAD

**Issue by Marigold** [childsish/dynamic-yaml#30]
Standard YAML supports merge keys like this
```python
import yaml
yaml.safe_load("""
a: &a
  b: 1
c:
  <<: *a
""")
```
`dynamic-yaml` unfortunately fails on the same sample
```python
import dynamic_yaml

dynamic_yaml.load("""
a: &a
  b: 1
c:
  <<: *a
""")
```
with error
```
ConstructorError: could not determine a constructor for the tag 'tag:yaml.org,2002:merge'
  in "<unicode string>", line 5, column 3:
      <<: *a
```

Would it be possible to add support for merge keys? We're using the library extensively in [our project](https://github.com/owid/etl) and find it very helpful, it'd be awesome to be able to do merges, too.

**Comment by childsish** [comment]
Turns out this was a relatively easy fix (4499770ce566d994f5e8c68a2889c87ca7b04325). Please try the 1.3.5

**Comment by Marigold** [comment]
Amazing, thank you @childsish!
